backend/tracim_backend/lib/core/content.py

```diff
@@ -1193,24 +1197,24 @@ def get_all_without_exception(
 
         return resultset.all()
 
-    def get_last_unread(
-        self, parent_id: int, content_type: str, workspace: Workspace = None, limit=10
-    ) -> typing.List[Content]:
-        assert parent_id is None or isinstance(parent_id, int)  # DYN_REMOVE
-        assert content_type is not None  # DYN_REMOVE
-        assert isinstance(content_type, str)  # DYN_REMOVE
-
-        read_revision_ids = self._session.query(RevisionReadStatus.revision_id).filter(
-            RevisionReadStatus.user_id == self._user_id
-        )
-
-        not_read_revisions = (
-            self._revisions_base_query(workspace)
-            .filter(~ContentRevisionRO.revision_id.in_(read_revision_ids))
-            .filter(ContentRevisionRO.workspace_id == Workspace.workspace_id)
-            .filter(Workspace.is_deleted.is_(False))
-            .subquery()
-        )
+    # def get_last_unread(
+    #     self, parent_id: int, content_type: str, workspace: Workspace = None, limit=10
+    # ) -> typing.List[Content]:
+    #     assert parent_id is None or isinstance(parent_id, int)  # DYN_REMOVE
+    #     assert content_type is not None  # DYN_REMOVE
+    #     assert isinstance(content_type, str)  # DYN_REMOVE
+    #
+    #     read_revision_ids = self._session.query(RevisionReadStatus.revision_id).filter(
+    #         RevisionReadStatus.user_id == self._user_id
+    #     )
+    #
+    #     not_read_revisions = (
+    #         self._revisions_base_query(workspace)
+    #         .filter(~ContentRevisionRO.revision_id.in_(read_revision_ids))
+    #         .filter(ContentRevisionRO.workspace_id == Workspace.workspace_id)
+    #         .filter(Workspace.is_deleted.is_(False))
+    #         .subquery()
+    #     )
```

Comment:
why this code is commented ? Delete it if no longueur required.

Reply:
unused, dropped.